ups/lsst_distrib.table

```diff
@@ -10,3 +10,4 @@ setupOptional(lsst_obs)
 setupOptional(synpipe)
 setupRequired(jointcal)
 setupRequired(verify)
+setupOptional(display_firefly)
```

Comment:
Why does this package need to be `setupOptional`?  I think @athornton would prefer that we are building binary packages for it.

Reply:
Ah, I believe you are right, it should be `setupRequired`. Let me go fix it...